test/forwarder/Forwarder.test.ts

```diff
@@ -341,7 +341,33 @@ contract('Forwarder', ([from]) => {
       const logs = await recipient.getPastEvents('TestForwarderMessage')
       assert.equal(logs.length, 1, 'TestRecipient should emit')
       assert.equal(logs[0].args.realSender, senderAddress, 'TestRecipient should "see" real sender of meta-tx')
-      assert.equal('1', (await fwd.getNonce(senderAddress)).toString(), 'verifyAndCall should increment nonce')
+      assert.equal('1', (await fwd.getNonce(senderAddress)).toString(), 'execute should increment nonce')
+    })
+
+    it('should revert if not given enough gas', async () => {
```

Comment:
as forwarder is a critical component, maybe add a call with 'value'
this makes it run a different, more interesting path (see previous comments)

Reply:
there is a separate test for value. as I said - a single [outer] wei too little crashes the entire TX.